File: src/main/java/com/lord/small_box/dtos/BigBagItemDto.java
package com.lord.small_box.dtos;

import java.math.BigDecimal;
import java.util.Calendar;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class BigBagItemDto {
	
	private Long id;
	
	private String code;
	
	private String measureUnit;
	
	private int quantity;
	
	private BigDecimal unitCost;
	
	private BigDecimal totalCost;
	
	private Calendar expirationDate;

}

File: front/small-box/src/app/models/bigBagItemDto.ts
export class BigBagItemDto{
   id!:number;
	
	 code!:string;
	
	 measureUnit!:string;
	
	 quantity!:number;

	 description!:string;
	
	 unitCost!:number;
	
	 totalCost!:number;
	
	 expirationDate!:Date;

	 depositControlId!:number;

	 bigBagId!:number;

	 

	 constructor(code:string,measureUnit:string,description:string,depositControlId:number){
		this.code = code;
		this.measureUnit = measureUnit;
		this.description = description;
		this.depositControlId = depositControlId;
	 }
}